docs/docusaurus.config.js

```diff
@@ -169,6 +169,13 @@ module.exports = {
         include: ["**/*.md", "**/*.mdx"], // Extensions to include.
       },
     ],
+    [
```

Comment:
this wasn't there earlier?

Reply:
We used to use the docusaurs-preset-classic's inbuilt UA integration earlier. But now we are using a standalone docusaurs plugin named plugin-google-gtag for the GA4.

Reply:
That's the reason I have added a new preset here.